crates/api/src/lib.rs

```diff
@@ -1582,6 +1670,162 @@ mod test {
         }
     }
 
+    // Creates a mock file containing JSON-LD of the ChronicleOperations
+    // that would be created by the given command, although not in any particular order.
+    fn test_create_agent_operations_import() -> assert_fs::NamedTempFile {
+        let file = assert_fs::NamedTempFile::new("import.json").unwrap();
+        assert_fs::prelude::FileWriteStr::write_str(
+            &file,
+            r#"
+        [
+            {
+                "@id": "_:n1",
+                "@type": [
+                "http://btp.works/chronicleoperations/ns#SetAttributes"
+                ],
+                "http://btp.works/chronicleoperations/ns#agentName": [
+                {
+                    "@value": "testagent"
+                }
+                ],
+                "http://btp.works/chronicleoperations/ns#attributes": [
+                {
+                    "@type": "@json",
+                    "@value": {}
+                }
+                ],
+                "http://btp.works/chronicleoperations/ns#domaintypeId": [
+                {
+                    "@value": "type"
+                }
+                ],
+                "http://btp.works/chronicleoperations/ns#namespaceName": [
+                {
+                    "@value": "testns"
+                }
+                ],
+                "http://btp.works/chronicleoperations/ns#namespaceUuid": [
+                {
+                    "@value": "6803790d-5891-4dfa-b773-41827d2c630b"
+                }
+                ]
+            },
+            {
+                "@id": "_:n1",
+                "@type": [
+                "http://btp.works/chronicleoperations/ns#CreateNamespace"
+                ],
+                "http://btp.works/chronicleoperations/ns#namespaceName": [
+                {
+                    "@value": "testns"
+                }
+                ],
+                "http://btp.works/chronicleoperations/ns#namespaceUuid": [
+                {
+                    "@value": "6803790d-5891-4dfa-b773-41827d2c630b"
+                }
+                ]
+            },
+            {
+                "@id": "_:n1",
+                "@type": [
+                "http://btp.works/chronicleoperations/ns#AgentExists"
+                ],
+                "http://btp.works/chronicleoperations/ns#agentName": [
+                {
+                    "@value": "testagent"
+                }
+                ],
+                "http://btp.works/chronicleoperations/ns#namespaceName": [
+                {
+                    "@value": "testns"
+                }
+                ],
+                "http://btp.works/chronicleoperations/ns#namespaceUuid": [
+                {
+                    "@value": "6803790d-5891-4dfa-b773-41827d2c630b"
+                }
+                ]
+            }
+        ]
+         "#,
+        )
+        .unwrap();
+        file
+    }
+
+    fn read_file_to_string(path: impl AsRef<std::path::Path>) -> String {
```

Comment:
Were you wanting `std::fs::read_to_string`?

Reply:
Yes!